### PR TITLE
NMS-14253: symlink latest version in project directories

### PR DESCRIPTION
**Pull request by RangerRick**
This PR should make "latest" links in each project directory.

Of course, I'm not sure it's working at the moment, since it dies from (I think) memory issues.  Sigh.

### CODE REVIEW

**Comment by RangerRick** [comment]
aha, I had some script issues with busybox, should be working now :)

**Review by Bonrob2 — COMMENTED** [review]
I don't have the expertise to provide a useful comment. Over to Ronny ...

**Comment by Bonrob2** [comment]
@indigo423 . Could you review and approve?

File: link-latest.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+set -e
+# shellcheck disable=SC3040
+set -o pipefail || :
+
+MYDIR="$(cd "$(dirname -- "$0")"; echo "$PWD")"
+cd "$MYDIR/public"
+
+find -- */* -name index.html | sed -e 's,/.*$,,' | sort -u | while read -r project; do
```
Comment (indigo423) [inline link-latest.sh:10]: The command `sort` can help with sorting version numbers:

```
cat test.txt| sort
0.0.1
1.0.1
10.0.10
10.0.2
11.0.100
11.1.100
11.2.1000
12.10.3
12.2.10
2.0.0
2.0.0
2.0.1
2.1.0
```
vs.

```
❯ cat test.txt| sort -V
0.0.1
1.0.1
2.0.0
2.0.0
2.0.1
2.1.0
10.0.2
10.0.10
11.0.100
11.1.100
11.2.1000
12.2.10
12.10.3
```

**Review by indigo423 — APPROVED** [review]
LGTM!

**Comment by Bonrob2** [comment]
Since Ben is away, merged with Veena's approval.

**Pull request merged**